docs/development/index.md

```diff
@@ -0,0 +1,131 @@
+| MH REST API v1 | /mediahaven-rest-api/    | [https://archief.viaa.be/mediahaven-rest-api/](https://archief.viaa.be/mediahaven-rest-api/)                         |
+| MH REST API v2 | /mediahaven-rest-api/v2/ | [https://archief.viaa.be/mediahaven-rest-api/v2/api-docs/](https://archief.viaa.be/mediahaven-rest-api/v2/api-docs/) |
+
+# API's
+
+Twee grote API's zijn beschikbaar voor de content partners:
+
+- REST API
+- OAI-PMH API
+
+Daar waar je via de REST API volledige _CRUD_-functionaliteit hebt ("_Create,
+Read, Update, Delete_"), is de OAI-PMH alleen-lezen. Anderzijds is een OAI-PMH
+API een welomlijnde standaard en is de REST API MediaHaven-specifiek.
+
+## REST API
+
+Momenteel zijn er twee versies van de REST API actief: versie 1 en versie 2.
+Het wordt aangeraden om, voor nieuwe integraties, meteen van start te gaan met
+versie 2.
+
+De documentatie van beide versies is hier te vinden:
+
+- REST API v1:
+  [https://archief.viaa.be/mediahaven-rest-api](https://archief.viaa.be/mediahaven-rest-api)
+- REST API v2:
+  [https://archief.viaa.be/mediahaven-rest-api/v2/api-docs/index.html](https://archief.viaa.be/mediahaven-rest-api/v2/api-docs/index.html)
+
+### Authenicatie voor de MediaHaven REST API v1
+
+Authenticatie op v1 is mogelijk via Basic Auth én via OAuth 2.0. Zie de
+documentatie voor meer info.
+
+### Authenicatie voor de MediaHaven REST API v2
+
+Authenicatie op v2 is **uitsluitend** mogelijk via OAuth 2.0!
+
+Voor de authorizatie zijn op de MediaHaven REST API v2 zijn momenteel twee
+OAuth2 flows geïmplementeerd:
+
+- “Authorization Code Grant Type”
+- “Resource Owner Credentials Grant Type”
+
+De keuze voor een flow hangt af van de use-case:
+
+- user-facing web applicatie → "Authorization Code Grant Type"
+- machine-to-machine applicatie integratie → "Resource Owner Credentials Grant
+  Type"
+
+(Zie bv [https://alexbilbie.com/guide-to-oauth-2-grants/](https://alexbilbie.com/guide-to-oauth-2-grants/))
+
+(De auth-flow die typisch gebruikt wordt voor machine-to-machine integraties is
+de "Client Credentials Grant Type". Deze is echter - nog - niet geïmplementeerd
+in de MediaHaven REST API).
+
+Om een applicatie via Oauth2 te laten authenticeren tegen MediaHaven, moet deze
+applicatie geregistreerd worden. We hebben dan van je nodig:
+
+- Organisatienaam
+- Naam van de integratie (of applicatie)
+- Korte beschrijving van de integratie
+- Destination server(s) (*)
+- Redirect-URL (*)
+
+(*) Enkel in het geval van de “Authorization Code Grant Type”.
+
+Je krijgt dan een "client_id" en een "client_secret". Om te authenticeren met
+deze client_id en client_secret, heb je een account nodig. Dat kan een
+persoonlijke account zijn of een "technische" account (bv. `mediahaven.api@<org>.be`)
+
+## OAI-PMH API
+
+Zie [https://developer.meemoo.be/docs/oai-pmh/]({{ site.baseurl }}{% link docs/oai-pmh/index.md %}) voor meer informatie.
```

Comment:
Should we put the REST API on a different page to align with the OAI-PMH?

Reply:
We should. Later. :wink: